source/content.css

```diff
@@ -822,15 +822,10 @@ body > .footer li a {
 }
 
 /* For 'extend-diff-expander' feature */
-.refined-github-diff-expand .js-expandable-line:hover .diff-expander,
-.refined-github-diff-expand .js-expandable-line:hover .blob-code {
+.js-expandable-line:hover .diff-expander,
```

Comment:
This prevents the css from being disabled with the feature

Reply:
Good point, but I'm not sure of why that would be necessary. If deemed necessary, a single `rgh-diff-expander` class can be added to `.diff-view` so all dynamically-added elements will be handled.